client/src/main/java/io/pravega/client/stream/TransactionalEventStreamWriter.java

```diff
@@ -26,17 +26,19 @@
      * Start a new transaction on this stream. This allows events written to the transaction be written an committed atomically.
      * Note that transactions can only be open for {@link EventWriterConfig#transactionTimeoutTime}.
      * 
+     * @throws TxnFailedException This exception could be thrown on beginTxn
```

Comment:
This kind of comments are not useful to the user

Reply:
Yes Enrico